pyccl/neutrinos.py

```diff
@@ -1,33 +1,61 @@
+import numpy as np
 from pyccl import ccllib as lib
+from pyccl.pyutils import check
 
-def Omeganuh2(a,Neff,mnu,TCMB):
+
+def Omeganuh2(a, Neff, mnu, TCMB=2.725):
     """Omeganuh2
     
     Args:
         a (float): Scale factor, normalized to 1 today.
         Neff (float): Number of relativistic neutrino species
         mnu (float): Neutrino mass (in eV)
-        TCMB (float): Temperature of the CMB (K)
+        TCMB (float, optional): Temperature of the CMB (K). Default: 2.725.
     Returns:
         Omeganuh2 (float or array_like)
```

Comment:
I think it would be ideal if we could include a comment in the documentation here which makes clear the difference between Omeganuh2 and ccl_omega_x called for neutrinos (they are not the same as z!=0 and I don't think this is obvious from the name).